### DISCUSSION TITLE
Appendix for the organization's 'readme'

### DISCUSSION THREAD

**Discussion by Ayush7614** [amplication/amplication#2694] [General]
👋 Welcome!

This discussion is meant to capture issues, questions, and requirements related to the implementation of the Appendix for the organization's 'readme' on the .github folder :)

To give you all a short overview that what this Appendix is for and how we can use .github folder, I write a short note here 👇

Currently, amplication org repo does not have the special readme in the "overview section" to display the required info about amplication and different projects/repo we have under the org, Adding an Appendix and glossary section will help new visitors to learn more about the project in a short amount of time.

To know about the .github you can follow [this](https://www.freecodecamp.org/news/how-to-use-the-dot-github-repository/) blog

I request all the contributors to drop their suggestions and thoughts on it under this discussion :)

**Comment by Ayush7614** [comment]
cc @yuval-hazaz

**Comment by yuval-hazaz** [comment]
Thanks @Ayush7614 
I created the .github repo, you are welcome to send a PR with your initial thoughts, and then we can follow up from there
